src/actions.js

```diff
@@ -0,0 +1,58 @@
+import axios from 'axios';
+import { store } from './App';
+
+export const setElementsData = (elementsData) => {
+  return {
+    type: 'SET_ELEMENTSDATA',
+    payload: elementsData,
+  }
+}
+
+const setLoadState = (isLoading) => {
```

Comment:
My sense is that this should not be a separate action. Why not just set the loading state as you are setting the data? Perhaps this will make more sense with the reducer refactor I'm suggesting in `reducer.js`